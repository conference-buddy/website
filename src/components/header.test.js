import React from "react";
import { render, screen, cleanup } from "@testing-library/react";
import Header from "./Header";

describe("Header", () => {
  let component;
  beforeAll(() => {
    component = render(<Header />);
  });

  afterAll(cleanup);

  it("renders the header element", () => {
    const header = screen.getByRole("banner");
    expect(header).toBeVisible();
  });

  it("renders all necessary elements", () => {
    expect(component.asFragment()).toMatchSnapshot();
  });
});
