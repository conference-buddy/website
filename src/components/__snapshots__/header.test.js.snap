// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders all necessary elements 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="MainNav px-sm-4 py-2 px-2 fixed-top d-flex justify-content-between MainNav-bg-out"
    >
      <div
        class="d-none d-sm-block d-flex align-items-center"
      >
        <a
          aria-current="page"
          class=""
          href="/"
        />
      </div>
    </nav>
  </header>
</DocumentFragment>
`;
